docs/reference_index.md

```diff
@@ -46,12 +46,19 @@ Input elements in HXML allow users to set local state on a Hyperview screen. Thi
 
 #### Style Elements
 
-Style elements in HXML define rules for the appearance of display and input elements.o
+Style elements in HXML define rules for the appearance of display and input elements.
 
 - [`<styles>`](/docs/reference_styles): Groups together all of the `<style>` rules for a screen.
 - [`<style>`](/docs/reference_style): A single style rule with unique id in the screen, can set multiple appearance properties.
 - [`<modifier>`](/docs/reference_modifier): Defines an appearance that should override the default for a style under certain local conditions, such as a user tap, selection, etc.
 
+#### Navigator Elements
+
+Navigator elements in HXML define a navigator hierarchy.
+
+- [`<navigator>`](/docs/reference_navigator): A definition of a stack or tab navigator.
```

Comment:
When I use the preview site, these links are 404s.

Reply:
Should be working now. You may need to run `yarn build` to see the updates.